data/azure/instance_types.json

```diff
@@ -133,8 +264,8 @@
   "Standard_H8": {"up": "Standard_H8m", "down": null, "superseded": null},
   "Standard_H8m": {"up": "Standard_H16", "down": "Standard_H8", "superseded": null},
   "Standard_H16": {"up": "Standard_H16m", "down": "Standard_H8m", "superseded": null},
-  "Standard_H16m": {"up": null, "down": "Standard_H16", "superseded": null},  
-  "Standard_H16r": {"up": "Standard_H16mr", "down": null, "superseded": null},
+  "Standard_H16m": {"up": "Standard_H16mr", "down": "Standard_H16", "superseded": null},  
+  "Standard_H16r": {"up": "Standard_H16mr", "down": "Standard_H16", "superseded": null},
```

Comment:
Can you cross-check the value for "down"

Reply:
Done